lib/round.rb

```diff
@@ -0,0 +1,52 @@
+class Round
```

Comment:
I wish I would have thought on getting all the frames in the class at once, instead of going one by one. Well thought as that make it more simple